src/lib/sort/sort-header.html

```diff
@@ -0,0 +1,20 @@
+<div class="mat-sort-header-container" type="button"
+     [class.mat-sort-header-position-before]="arrowPosition == 'before'">
+  <button class="mat-sort-header-button"
```

Comment:
Ah misunderstood, thought this was an accessibility thing, putting type on the wrong element. Good now